lib/m2ts/caption-stream.js

```diff
@@ -186,8 +229,8 @@
       ROLL_UP_2_ROWS             = 0x1425,
       ROLL_UP_3_ROWS             = 0x1426,
       ROLL_UP_4_ROWS             = 0x1427,
+      RESUME_DIRECT_CAPTIONING   = 0x1429,
```

Comment:
Yes, I thought about removing it but I get the sneaking suspicion that we will be using it someday pretty soon.